docs/algorithms/gail.rst

```diff
@@ -29,35 +29,47 @@ Detailed example notebook: :doc:`../tutorials/3_train_gail`
     from imitation.data import rollout
     from imitation.data.wrappers import RolloutInfoWrapper
     from imitation.policies.serialize import load_policy
-    from imitation.rewards.reward_nets import BasicRewardNet
+    from imitation.rewards.reward_nets import BasicShapedRewardNet
     from imitation.util.networks import RunningNorm
     from imitation.util.util import make_vec_env
 
-    rng = np.random.default_rng(0)
+    SEED = 42
 
     env = make_vec_env(
         "seals/CartPole-v0",
-        rng=rng,
-        n_envs=8,
-        post_wrappers=[lambda env, _: RolloutInfoWrapper(env)],  # for computing rollouts
+        rng=np.random.default_rng(SEED),
+        n_envs=5,
```

Comment:
Why 5? Why do we make `env` twice here?

Reply:
not sure where this came from :sweat_smile:   but it's not needed. fixed and reuses the same env now